tests/suites/test_suite_psa_crypto.data

```diff
@@ -1410,11 +1390,6 @@ depends_on:MBEDTLS_CIPHER_MODE_CTR
 # Either INVALID_ARGUMENT or NOT_SUPPORTED would be reasonable here
 cipher_setup:PSA_KEY_TYPE_RAW_DATA:"000102030405060708090a0b0c0d0e0f":PSA_ALG_CTR:PSA_ERROR_NOT_SUPPORTED
 
-PSA cipher setup: incompatible key ARC4 for CTR
```

Comment:
We still need to have a test with an incompatible key type. Please keep this test case with CHACHA key instead of ARC4. The key size must change to 32 bytes.